app/tests/groups_tests/test_views.py

```diff
@@ -81,3 +89,106 @@ def get_user_autocomplete():
         )
         assert response.status_code == 302
         assert not group.user_set.filter(pk=u.pk).exists()
+
+
+@pytest.mark.django_db
+class TestAutocompleteViews:
+    @pytest.mark.parametrize(
+        "is_verified", (False, True),
+    )
+    @pytest.mark.parametrize(
+        "filter", (("username"), ("email"), ("full_name"),),
+    )
+    def test_autocomplete_filter_options(self, client, filter, is_verified):
+        archive = ArchiveFactory()
+
+        admin = UserFactory()
+        archive.add_editor(admin)
+        first_name = "Jane"
+        last_name = "Doe"
+
+        if is_verified:
+            call_command("check_permissions")
```

Comment:
`check_permissions` should only be called once at test setup (outside of the individual tests), why is it being called again here?

Reply:
I'm not sure I understand what you mean by calling it at test setup. We put it here because it seemed necessary to create a verified user. Where should it go instead?

Reply:
Ah yes, I took a look at the verification tests and it's called there too (https://github.com/comic/grand-challenge.org/blob/master/app/tests/verification_tests/test_forms.py#L85), it _shouldn't_ be necessary, but looks like it is. We don't need to solve that here. OK to leave this as it.